addons/event/controllers/main.py

```diff
@@ -23,3 +26,58 @@ def event_ics_file(self, event, **kwargs):
             ('Content-Length', len(content)),
             ('Content-Disposition', content_disposition('%s.ics' % event.name))
         ])
+
+    @route(['/event/my_tickets/<int:event_id>'], type='http', auth='public')
```

Comment:
Note that we prefer a "drawer" naming, like ``/event/<id>/my_tickets`` to know we are under a given event.